source/neuropod/bindings/java/src/test/java/com/uber/neuropod/PythonStringsModelTest.java

```diff
@@ -1,4 +1,4 @@
-/* Copyright (c) 2020 UATC, LLC
+/* Copyright (c) 2022 The Neuropod Authors
```

Comment:
(note that this date was changed from `2020` to `2022` because the file was actually created in 2022, but I listed the wrong date in the original commit)